### DISCUSSION TITLE
Personnalisation du texte d'un type customfile pour les formats acceptés

### DISCUSSION THREAD

**Discussion by ageliinas** [MTESSDev/FRW#139] [Annonces]
Bonjour,

Nous utilisons la propriété de téléverser des fichiers, et nous avons mis seulement les formats png, jpg et pdf de valide avec  mime: image/jpeg,image/png,application/pdf. Cela fonctionne très bien. Par contre, le petit texte qui semble être géré par le type customfile spécifie les formats autorisés. 

![Capture2](https://github.com/MTESSDev/FRW/assets/80847091/383c0394-0af9-41e2-8fee-189bf55d61f0)

Ce petit texte n'est pas dans notre code, je ne peux donc pas le modifier et dire que c'est seulement les formats png, jpg et pdf qui sont acceptés (et non les formats png, jpg, pdf, docx ou odt comme il est écrit présentement). Est-ce qu'il existe un moyen de pouvoir personnalisé ce texte, ou peut-être le supprimer pour qu'on puisse écrire le notre? 

Merci,
Bonne journée

**Comment by j0ber** [comment]
Bonjour,
Il faut utiliser la propriété "help".

https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=controlesBase
https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionDivers  (voir exemple avec précision)

Je vais créer un issue afin que nous ajoutions un exemple dans notre page qui traite des fichiers. 

```YAML
help:
    fr: Exemple de précision

**Comment by ageliinas** [comment]
On ne savait pas que ca remplaçait le texte par défaut aussi, merci!
